test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -436,76 +436,80 @@ describe('View recordset,', function () {
+                    const exportDropdown = chaisePage.recordsetPage.getExportDropdown();
+                    exportDropdown.click().then(function () {
+                        const csvOption = element(by.partialLinkText('Search results (CSV)'));
+                        expect(csvOption.getText()).toBe("Search results (CSV)");
+                        return csvOption.click();
+                    }).then(function () {
+                        browser.wait(function() {
+                            return fs.existsSync(process.env.PWD + "/test/e2e/Accommodations.csv");
+                        }, browser.params.defaultTimeout).then(function () {
+                            done();
+                        }, function () {
+                            expect(false).toBeTruthy("Accommodations.csv was not downloaded");
+                        });
+                    }).catch(function (err) {
+                        console.log(err);
+                        done.fail();
+                    });
+                });
 
-            //     it("should have 'BDBag' as a download option and download the file.", function(done) {
-            //         chaisePage.recordsetPage.getExportDropdown().click().then(function () {
-            //             var bagOption = chaisePage.recordsetPage.getExportOption("BDBag");
-            //             expect(bagOption.getText()).toBe("BDBag");
-            //             return bagOption.click();
-            //         }).then(function () {
-            //             return chaisePage.waitForElement(chaisePage.recordsetPage.getExportModal());
-            //         }).then(function () {
-            //             return chaisePage.waitForElementInverse(chaisePage.recordsetPage.getExportModal());
-            //         }).then(function () {
-            //             return browser.wait(function() {
-            //                 return fs.existsSync(process.env.PWD + "/test/e2e/accommodation.zip");
-            //             }, browser.params.defaultTimeout);
-            //         }).then(function () {
-            //             chaisePage.waitForElementInverse(element(by.css(".export-progress")));
-            //             done();
-            //         }).catch(function (err) {
-            //             done.fail(err);
-            //         });
-            //     });
-            // }
+                it("should have 'BDBag' as a download option and download the file.", function(done) {
+                    const exportDropdown = chaisePage.recordsetPage.getExportDropdown();
+                    let modalDialog;
+                    exportDropdown.click().then(function () {
+                        const bagOption = element(by.partialLinkText('BDBag'));
+                        expect(bagOption.getText()).toBe("BDBag");
+                        return bagOption.click();
+                    }).then(function () {
+                        modalDailog = element(by.css('.modal-dialog'));
```

Comment:
This selector doesn't work as I expect it to. We used a selector that was specifically for the export modal before, so we should fix why that selector isn't working (add `export-progress` class to the `<Modal ...>` in `export-modal.tsx`.